python-packages/order_utils/src/zero_ex/order_utils/__init__.py

```diff
@@ -9,3 +9,145 @@
 --networkId 50 -m "concert load couple harbor equip island argue ramp clarify
 fence smart topic"``.
 """
+
+import json
+from typing import Dict
+from pkg_resources import resource_string
+
+from mypy_extensions import TypedDict
+
+from eth_utils import is_address, keccak, to_checksum_address, to_bytes
+from web3 import Web3
+from web3.utils import datatypes
+import web3.exceptions
+
+
+EXCHANGE_ABI = json.loads(
+    resource_string("zero_ex.contract_artifacts", "artifacts/Exchange.json")
+)["compilerOutput"]["abi"]
+
+NETWORK_TO_EXCHANGE_ADDR: Dict[str, str] = {
+    "1": "0x4f833a24e1f95d70f028921e27040ca56e09ab0b",
+    "3": "0x4530c0483a1633c7a1c97d2c53721caff2caaaaf",
+    "42": "0x35dd2932454449b14cee11a94d3674a936d5d7b2",
+    "50": "0x48bacb9266a570d521063ef5dd96e61686dbe788",
+}
+
+NULL_ADDRESS = "0x0000000000000000000000000000000000000000"
+
+
+class Order(TypedDict):  # pylint: disable=too-many-instance-attributes
```

Comment:
Is this `TypedDict` kosher?

Reply:
Talked offline and resolved all concerns.